apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-applications/vo-applications.component.html

```diff
@@ -18,10 +18,32 @@ <h1 class="page-subtitle">{{'VO_DETAIL.APPLICATION.TITLE' | translate}}</h1>
       <mat-option value="rejected">{{'VO_DETAIL.APPLICATION.SELECTION_REJECTED' | translate}}</mat-option>
     </mat-select>
   </mat-form-field>
+  <mat-form-field class="input-width-250 mr-2">
+    <mat-label>{{'VO_DETAIL.APPLICATION.TABLE_GROUPS_SELECTION' | translate}}</mat-label>
+    <mat-select (selectionChange)="select()" [(value)]="selectionGroups" disableOptionCentering>
+      <mat-option value="only_vo">{{'VO_DETAIL.APPLICATION.SELECTION_ONLY_VO' | translate}}</mat-option>
+      <mat-option value="also_groups">{{'VO_DETAIL.APPLICATION.SELECTION_ALSO_GROUPS' | translate}}</mat-option>
+    </mat-select>
+  </mat-form-field>
   <app-debounce-filter
     [placeholder]="'VO_DETAIL.APPLICATION.FILTER'"
     (filter)="applyFilter($event)">
   </app-debounce-filter>
+  <div>
+    <h1 class="page-subtitle-2">{{'VO_DETAIL.APPLICATION.DATE_TITLE' | translate}}</h1>
+    <mat-form-field class="space-right">
+      <mat-label>{{'VO_DETAIL.APPLICATION.DATE.START' | translate}}</mat-label>
+      <input matInput [matDatepicker]="datePickerStart" [min]="minDate" [max]="maxDate" [formControl]="startDate" (dateChange)="dateChange()">
+      <mat-datepicker-toggle [for]="datePickerStart" matSuffix></mat-datepicker-toggle>
+      <mat-datepicker #datePickerStart></mat-datepicker>
+    </mat-form-field>
+    <mat-form-field>
+      <mat-label>{{'VO_DETAIL.APPLICATION.DATE.END' | translate}}</mat-label>
+      <input matInput [matDatepicker]="datePickerEnd" [min]="minDate" [max]="maxDate" [formControl]="endDate" (dateChange)="dateChange()">
+      <mat-datepicker-toggle [for]="datePickerEnd" matSuffix></mat-datepicker-toggle>
+      <mat-datepicker #datePickerEnd></mat-datepicker>
+    </mat-form-field>
```

Comment:
please create reusable component for datepicker that would copy the behaviour that is in the sponsored members page - dialog for creating 
![image](https://user-images.githubusercontent.com/32871167/106305008-6a029700-625c-11eb-934c-799fbfdcbdc5.png)
create new component that you will use here and in the sponsored members and copy their behaviour

Reply:
I created separate component for date range and I added new condition which doesn't allow to set older end date than start date and newer start date than end date. I also replaced Group/VO applications selection with checkbox.